admin/bin/qserv-admin.py

```diff
@@ -169,6 +169,9 @@ def __init__(self, connInfo):
     RESTORE <inFile>;
     SHOW DATABASES;
     SHOW NODES;
+    CREATE CHUNK <dbName>.<tableName> <chunk> <nodeName>;
+    DELETE CHUNK <dbName>.<tableName> <chunk> <nodeName>;
```

Comment:
You're right! The node name should not be there (in the 'DELETE CHUNK' operation). I will remove it.